pom.xml

```diff
@@ -117,8 +117,8 @@
   <properties>
     <maven.compiler.release>8</maven.compiler.release>
     <!-- Set to same version as release target for consistency -->
-    <maven.compiler.source>1.${maven.compiler.release}</maven.compiler.source>
-    <maven.compiler.target>1.${maven.compiler.release}</maven.compiler.target>
+    <maven.compiler.source>${maven.compiler.release}</maven.compiler.source>
```

Comment:
I think we can fully remove those properties, as we build with Java11+ only

Reply:
They were originally added as some tools and IDEs (IDEA) did not pick up `maven.compiler.release`, so just to "fully advertise" project is 8. We can remove them, but there is no harm in them either.